CockpitCore/src/main/java/com/polidea/cockpit/core/type/CockpitListType.kt

```diff
@@ -1,4 +1,8 @@
 package com.polidea.cockpit.core.type
 
 
-data class CockpitListType<T: Any>(val items: List<T>, var selectedIndex: Int)
\ No newline at end of file
+data class CockpitListType<T: Any>(val items: List<T>, var selectedIndex: Int) {
+    fun getSelectedItem(): T {
+        return items[selectedIndex]
+    }
```

Comment:
Expression body is nice and concise
`fun getSelectedItem() = items[selectedIndex]`